packages/block-editor/src/components/link-control/is-url-like.js

```diff
@@ -20,6 +15,6 @@ import { isURL } from '@wordpress/url';
  * @return {boolean} whether or not the value is potentially a URL.
  */
 export default function isURLLike( val ) {
-	const isInternal = startsWith( val, '#' );
+	const isInternal = val.startsWith( '#' );
```

Comment:
I think we should handle nullish value as lodash's function did..
```suggestion
	const isInternal = val?.startsWith( '#' );
```

Reply:
Sure, happy to play on the safe side 👍